### DISCUSSION TITLE
关于 COMP Exchange

### DISCUSSION THREAD

**Discussion by David200308** [ZHANG-CAIQI/PolyU_COMP#55] [COMP Tips]
2022复旦大学暑假交换（通过 PolyU 申请）：（申请GPA：3.0+）

关于CS的课程: 
Introduction to AI – Probabilistic Reasoning and Decision Making （UCSD 教授教授 - Prof. Paul Cao）
这门课是关于人工智能的数学基础课，讲述了人工智能网络背后的概率数学论和模型

**Comment by caiqizh** [comment]
这个课是在exchange期间上的意思咩？

**Reply by David200308** [reply]
对滴
